src/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -80,6 +80,42 @@
                               PreferredPackages="$(Id);@(RuntimeDependency)" />
   </Target>
 
+  <UsingTask TaskName="GetVersionFromAssembly" AssemblyFile="$(LocalBuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.Local.dll"/>
+
+  <Target Name="GenerateHashVersionsFile" BeforeTargets="GetPackageFiles">
+    <PropertyGroup>
+      <_SystemRuntimePath Condition="$([System.String]::new('%(File.Identity)').ToLowerInvariant().Contains('system.runtime.dll'))">%(File.Identity)</_SystemRuntimePath>
+      <_SystemPrivateCoreLibPath Condition="$([System.String]::new('%(File.Identity)').ToLowerInvariant().Contains('system.private.corelib.dll'))">%(File.Identity)</_SystemPrivateCoreLibPath>
+      <_SkipCoreClrHash Condition="'$(_SystemPrivateCoreLibPath)' == ''">true</_SkipCoreClrHash>
```

Comment:
When building the identity package, coreclr binaries won't be in the `File` item.